src/main/java/org/folio/search/converter/ConsortiumHoldingMapper.java

```diff
@@ -0,0 +1,23 @@
+package org.folio.search.converter;
+
+import org.folio.search.domain.dto.ConsortiumHolding;
+import org.folio.search.domain.dto.Holding;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
is it needed to have this as a bean? seems like we can just have it as a utility class (and also ConsortiumItemMapper)